packages/keyring-eth-simple/src/simple-keyring.test.ts

```diff
@@ -21,8 +21,10 @@ import {
 import { add0x, Hex } from '@metamask/utils';
 import assert from 'assert';
 import { keccak256 } from 'ethereum-cryptography/keccak';
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
What is the naming convention error here? Asking because I don't see a relevant difference with the imports from other files

Reply:
I believe that's because "default" exports should be named using camelCase naming convention, but here it's using PascalCase instead.

You are allowed to use PascalCase if you use "destructuring import" (not sure what's the official name for this).